precompile/contracts/warpmessenger/contract.go

```diff
@@ -211,22 +206,10 @@ func getBlockchainID(accessibleState contract.AccessibleState, caller common.Add
 	return packedOutput, remainingGas, nil
 }
 
-// UnpackGetVerifiedWarpMessageInput attempts to unpack [input] into the *big.Int type argument
-// assumes that [input] does not include selector (omits first 4 func signature bytes)
-func UnpackGetVerifiedWarpMessageInput(input []byte) (*big.Int, error) {
-	res, err := WarpMessengerABI.UnpackInput("getVerifiedWarpMessage", input)
-	if err != nil {
-		return big.NewInt(0), err
-	}
-	unpacked := *abi.ConvertType(res[0], new(*big.Int)).(**big.Int)
-	return unpacked, nil
-}
-
-// PackGetVerifiedWarpMessage packs [messageIndex] of type *big.Int into the appropriate arguments for getVerifiedWarpMessage.
-// the packed bytes include selector (first 4 func signature bytes).
+// PackGetVerifiedWarpMessage packs the include selector (first 4 func signature bytes).
```

Comment:
Can you clarify this comment?

Reply:
Went one better and removed this function all together since it wasn't used anywhere. :sweat_smile: Same with `PackGetBlockchainID()`.